extensionsGallery.json

```diff
@@ -1740,14 +1740,14 @@
 					},
 					"versions": [
 						{
-							"version": "0.0.1",
-							"lastUpdated": "5/31/2019",
+							"version": "0.0.2",
```

Comment:
Why are we updating all these other ones? For example admin-tool-ext was only updated to use the new iKey but that's not something we need for the October release. Same for some of the other ones.

Reply:
yeah, I was trying to make the insiders match the stable, but realized these changes may not be ready.  I move out the ones that don't have the functional changes we want to publish.